RNPermissions.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
   s.authors                   = package["author"]
   s.homepage                  = package["homepage"]
 
-  s.ios.deployment_target     = "10.0"
+  s.ios.deployment_target     = "12.0"
```

Comment:
to avoid break changes, I think, it should be like this `s.ios.deployment_target =(fabric_enabled ? "12.0" : "10.0")`

Reply:
Sure, yeah this seems reasonable. I didn't understand that this only affects new architecture (still not sure why).